### DISCUSSION TITLE
I can't open canvas..

### DISCUSSION THREAD

**Discussion by Akk1kuroneko0** [thatskymod/Sky-CotL-Scripts#52] [Q&A]
I was having problems with canvas, didn't know what to do. 

But checking upon, there is something wrong with smite.. :(

**Comment by Sagimoster** [comment]
Tha same problem to me

**Comment by thatskymod** [comment]
Live crash fixed in [libsmite-1.0.7d.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libsmite-1.0.7d.so)

**Comment by ItruthHurtsI** [comment]
For me the canvas app keeps stopping.. and I've noticed this problem happens after I updated my game. What shall I do? It just keeps stopping I can't open it even a second
